submissions/api.py

```diff
@@ -89,6 +90,15 @@ def __init__(self, msg="", field_errors=None):
         )
 
 
+# For API stability and low memory usage, large result sets are returned using namedtuples.
+# They take up only as much memory as tuples, but fields are accessible by name,
+# so we can change the fields in the future if needed
+LatestSubmission = namedtuple(
```

Comment:
LatestSubmission seems odd as a class name, given that it's just a submission. If we want to differentiate it from the model, I guess we could call it a SubmissionTuple or something.